### DISCUSSION TITLE
Luciano Ponce

### DISCUSSION THREAD

**Discussion by ghost** [Launch-X-Latam/OnBoarding#10] [General]
Hola Soy luciano Ponce No me pude Conectar recien estoy entrando

**Comment by JosafatJimenezB** [comment]
Descuida, las sesiones siempre se quedan grabadas y puedes acceder desde la pestaña de grabaciones o desde el link mismo de la sesión

saludos!

**Reply by Gl096** [reply]
Hola Josafat ¿en dónde puedo ver las grabaciones? gracias

**Reply by JosafatJimenezB** [reply]
En la parte de canales te diriges a general, estando ahi te vas a las parte superior donde dice grabaciones y ahi estaran las sesiones grabadas

**Reply by ghost** [reply]
Holaaa estoy ,uyyy desorientado no se como ingresar a la plataforma ya que la Pagina LaunchXLAtam dice que esta llena...me podrian acesorar??
